frontend/src/views/WheresMyGene/components/HeatMap/style.ts

```diff
@@ -1,45 +1,22 @@
 import styled from "@emotion/styled";
 import { X_AXIS_CHART_HEIGHT_PX, Y_AXIS_CHART_WIDTH_PX } from "./utils";
 
+export const CHART_LEFT_PADDING = 10;
+
 export const Container = styled.div`
   height: 75vh;
   width: 100%;
   overflow: scroll;
   position: relative;
 `;
 
-export const XAxisWrapper = styled.div`
-  ${xAxisWidth}
-
-  background-color: white;
-  height: ${X_AXIS_CHART_HEIGHT_PX}px;
-  position: sticky;
-  top: 0;
-  z-index: 2;
-`;
-
-export const XAxisMask = styled.div`
-  background-color: white;
-  height: ${X_AXIS_CHART_HEIGHT_PX}px;
-  width: ${Y_AXIS_CHART_WIDTH_PX}px;
-  position: sticky;
-  left: 0;
-`;
-
-export const XAxisContainer = styled.div`
-  ${xAxisWidth}
-
-  background-color: white;
-  height: ${X_AXIS_CHART_HEIGHT_PX}px;
-  position: absolute;
-`;
-
 export const YAxisWrapper = styled.div`
   width: ${Y_AXIS_CHART_WIDTH_PX}px;
-  position: sticky;
```

Comment:
I think we lost sticky y axis unless we no longer need it?
![demo](https://user-images.githubusercontent.com/6309723/210645636-5ff800c6-8f73-424f-b485-e4ebb51964cd.gif)

Reply:
Good catch!